### ISSUE TITLE
Nested lists rendering

### ISSUE THREAD

**Issue by villanella** [doorstop-dev/doorstop#601]
Doorstop-server can't render nested lists without empty lines.

For example,

    1. Item 1
    2. Item 2
      2.1. Item 2.1
        2.1.1. Item 2.1.1
    3. Item 3

This is rendered as:

    1. Item 1
    2. Item 2 2.1. Item 2.1 2.1.1. Item 2.1.1
    3. Item 3

All nested items are concatenated into one line.

Yet another example:

```
1. Item 1
    1. Item 1.1
    2. Item 1.2
2. Item 2
    1. Item 2.1
```
      
This nested items are rendered as simple, not nested, list:
```
1. Item 1
2. Item 1.1
3. Item 1.2
4. Item 2
5. Item 2.1
```

Expected behavior: support of nested lists made by indents and numbers.
Workaround: to use extra empty lines between items.

Used version - Doorstop v3.0b4 (markdown is default itemformat).

**Comment by JustinW80** [comment]
I think this might be related to #589, where the template theme is missing from versions after v3.0b4.

**Comment by villanella** [comment]
The assets folder exists, the result is the same in my v3.0b4 version and in stable version.

**Comment by neerdoc** [comment]
I'm working on a fix for this now. Please note though, that _correctly_ numbered lists in markdown requires four (4) spaces as indentation, so the first example does not work even for markdown publishing.

**Comment by jacebrowning** [comment]
This beta release contains a fix for this issue: https://pypi.org/project/doorstop/3.0b11/ cc @neerdoc

**Comment by neerdoc** [comment]
Fixed by #615
